### PR TITLE
Improve javadocs for querydsl-core

### PR DESCRIPTION
**Pull request by timowest**
related to https://github.com/querydsl/querydsl/issues/1190

### CODE REVIEW

File: querydsl-core/src/main/java/com/querydsl/core/FetchableQuery.java [hunk 1]
```diff
@@ -41,8 +45,8 @@
      * Apply the given transformer to this Projectable instance and return the results
```
Comment (Shredder121) [inline querydsl-core/src/main/java/com/querydsl/core/FetchableQuery.java]: Search+replace `Projectable` => `Fetchable` right?

Comment (timowest) [inline querydsl-core/src/main/java/com/querydsl/core/FetchableQuery.java]: right ;)

File: querydsl-core/src/main/java/com/querydsl/core/FetchableQuery.java [hunk 2]
```diff
@@ -23,26 +25,28 @@
...
-     * @param exprs
-     * @return
+     * @param exprs new projection
+     * @return this
      */
     FetchableQuery<Tuple, ?> select(Expression<?>... exprs);
 
     /**
-     * Apply the given transformer to this Projectable instance and return the results
+     * Apply the given transformer to this Fetchable instance and return the results
```
Comment (Shredder121) [inline querydsl-core/src/main/java/com/querydsl/core/FetchableQuery.java]: I think this should actually be `{@code FetchableQuery}`, my bad

File: querydsl-core/src/main/java/com/querydsl/core/support/FetchableQueryBase.java [hunk 3]
```diff
@@ -26,7 +26,7 @@
 import com.querydsl.core.types.SubQueryExpression;
 
 /**
- * ProjectableQuery extends the {@link QueryBase} class to provide default
+ * {@code FetchableQueryBase} extends the {@link FetchableQueryBase} class to provide default
```
Comment (Shredder121) [inline querydsl-core/src/main/java/com/querydsl/core/support/FetchableQueryBase.java]: It still extends QueryBase.

**Pull request merged**